src/test/java/com/training/assignments/StudentTest.java

```diff
@@ -0,0 +1,23 @@
+package com.training.assignments;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class StudentTest {
+
+    @Test
+    public void testStudent()
+    {
+        Student student=new Student();
+        student.setName("Naviya");
+        student.setLastName("Dayanand");
+        student.setRollNo(1);
+        Student.setSchoolName("CJCS");
+
+        Assertions.assertEquals("Naviya",student.getName());
```

Comment:
Add error message in Assertion example shown below.
  Assertions.assertEquals(,student.getName(),name,"Name doesn't match");